cypress/e2e/michael.cy.ts

```diff
@@ -4,18 +4,66 @@ describe('obstacle practice', () => {
         cy.contains('a', 'Click Me 2x').click()
         cy.contains('a', 'Click me once more').click()
         cy.contains('Good job!')
-    });
+    })
 
     it('41041-input', () => {
         cy.visit('https://obstaclecourse.tricentis.com/Obstacles/41041')
         cy.get('input[name="resulttext"]')
             .type('{Click}', { parseSpecialCharSequences: false })
         cy.contains('Good job!')
-    });
+    })
 
     it('60469-drag', () => {
         cy.visit('https://obstaclecourse.tricentis.com/Obstacles/60469')
         cy.get('#toscabot').dragTo('#to')
         cy.contains('Good job!')
     })
+
+    it('41038', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/41038')
+        cy.contains('Click into my right half').click('right')
+        cy.contains('Good job!')
+    })
+
+    it('22505', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/22505')
+        cy.contains('Click me!').click()
+        cy.contains('Good job!')
+    })
+    it.only('12952', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/12952')
+        cy.get('a#id').eq(1).click()
```

Comment:
This is not right either. Can you look up the element for its parent / ancestor node?